docs/ref/contrib/gis/gdal.txt

```diff
@@ -1817,6 +1828,67 @@ Here's how to create a raster and return it as a file in an
     ... })
     >>> HttpResponse(rast.vsi_buffer, 'image/tiff')
 
+Using other Virtual Filesystems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Depending on the local build of GDAL other virtual filesystems might be
+supported and can be used by prepending the provided path with the appropriate
+``/vsi*/`` prefix.
+
+.. warning:
+
+    Rasters with names starting with `/vsi*/` will be treated as rasters from
+    the GDAL virtual filesystems. Django doesn't perform any extra validation.
+
+The following sections provide some examples on how to use ``vsi*``
+filesystems, a full reference of each filesystem, alongside possible GDAL
+configuration options, can be found in `GDAL Virtual Filesystems
+documentation.`__
+
+__ https://gdal.org/user/virtual_file_systems.html
+
+Compressed rasters
+^^^^^^^^^^^^^^^^^^
+
+Instead decompressing the file and instantiating the resulting raster, GDAL can
+directly access compressed files using the ``/vsizip/``, ``/vsitar/`` or
```

Comment:
Comma is missing before 'or'/'and' (see https://en.wikipedia.org/wiki/Serial_comma).
```suggestion
directly access compressed files using the ``/vsizip/``, ``/vsitar/``, or
```